frontend/vue.config.js

```diff
@@ -0,0 +1,17 @@
+/*const { defineConfig } = require('@vue/cli-service')
+module.exports = defineConfig({
+  transpileDependencies: true
+})
+*/
+module.exports = {
+  devServer: {
+    port: 3000,
```

Comment:
Es wäre super, wenn Variablen wie port oder die target Url etc. in eine extra Config Datei ausgelagert werden. Dass erleichtert das Deployen auf den Production Server.